Python-Files/day16_OOP_turtle.py

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
Worked with the turtle imports and packages. It is nice that Pycharm gives you the ability to search and add packages within the IDE itself, although using pip install in the terminal is quick if you know what you need